source/renderer/assets/context/tabs.json

```diff
@@ -0,0 +1,21 @@
+[
+  {
+    "label": "menu.rename_file",
+    "command": "file-rename"
+  },
+  {
+    "label": "menu.delete_file",
+    "command": "file-delete"
+  },
+  {
+    "label": "menu.duplicate_file",
+    "command": "file-duplicate"
+  },
+  {
+    "type": "separator"
+  },
+  {
+    "label": "menu.close_all",
+    "command": "file-close-all"
+  }
+]
```

Comment:
Nitpick: EOF Newline here please.

Reply:
Well there is indeed a new line at the end of the file as shown by the file hexdump below. I am guessing Github diff view doesn't show that?

```bash
hexdump -c  source/renderer/assets/context/tabs.json
0000000   [  \n           {  \n                   "   l   a   b   e   l
0000010   "   :       "   m   e   n   u   .   r   e   n   a   m   e   _
0000020   f   i   l   e   "   ,  \n                   "   c   o   m   m
0000030   a   n   d   "   :       "   f   i   l   e   -   r   e   n   a
0000040   m   e   "  \n           }   ,  \n           {  \n            
0000050       "   l   a   b   e   l   "   :       "   m   e   n   u   .
0000060   d   e   l   e   t   e   _   f   i   l   e   "   ,  \n        
0000070           "   c   o   m   m   a   n   d   "   :       "   f   i
0000080   l   e   -   d   e   l   e   t   e   "  \n           }   ,  \n
0000090           {  \n                   "   l   a   b   e   l   "   :
00000a0       "   m   e   n   u   .   d   u   p   l   i   c   a   t   e
00000b0   _   f   i   l   e   "   ,  \n                   "   c   o   m
00000c0   m   a   n   d   "   :       "   f   i   l   e   -   d   u   p
00000d0   l   i   c   a   t   e   "  \n           }   ,  \n           {
00000e0  \n                   "   t   y   p   e   "   :       "   s   e
00000f0   p   a   r   a   t   o   r   "  \n           }   ,  \n        
0000100   {  \n                   "   l   a   b   e   l   "   :       "
0000110   m   e   n   u   .   c   l   o   s   e   _   a   l   l   _   t
0000120   a   b   s   "   ,  \n                   "   c   o   m   m   a
0000130   n   d   "   :       "   f   i   l   e   -   c   l   o   s   e
0000140   -   a   l   l   "  \n           }  \n   ]  \n                
000014c

```